### ISSUE TITLE
Implement Chat

### ISSUE THREAD

**Issue by geleeroyale** [MetaFam/interspace.chat#2]
1 - The idea is to pull in a React compatible IRC client - (need to deploy Server)

2 - In a second phase the IRC rooms shall be bridged to corresponding Discord rooms

**Comment by Christofon** [comment]
I started to deploy a IRC Droplet on digitalocean. Not sure if that's the best solution, someone got a better idea?Regarding the react compatible client I found a few options but nothing labeled as react package specifically.

**Comment by geleeroyale** [comment]
We should talk about deployments for MetaGame very soon, as this will become a thing and if you keep it running, costs will accumulate of course. @peth-yursick @hammadj

**Comment by peth-yursick** [comment]
Yes! Would you like to keep covering it for a while still, and have it count as an investment, or get reimbursed for everything so far and going forward?

**Comment by hammadj** [comment]
@geleeroyale Right now we have free deployments from Netlify. If we need other servers for anything, we can definitely reimburse costs from the funding pool. Ideally we would use as much free-tier stuff as long as possible.

Why can't we just implement the chat through discord API directly? Interspace could just be a lightweight discord client for a specific channel maybe

**Comment by geleeroyale** [comment]
Already have this working on https://staging.interspace.chat - will commit back

**Comment by pakokrew** [comment]
What do you think about using 3Box chat ? 
https://docs.3box.io/api/messaging

**Comment by peth-yursick** [comment]
3Box chat is cool, but would rather keep it all in one place (Discord)

**Comment by geleeroyale** [comment]
Will use a discord widget for this - expected to be in METAFEST pre-release
